server/services/overviewPageService.test.ts

```diff
@@ -27,24 +33,42 @@ describe('OverviewPageService', () => {
     getAdjudications: jest.fn(async () => adjudicationSummaryMock),
     getAccountBalances: jest.fn(async () => accountBalancesMock),
     getAssessments: jest.fn(async () => assessmentsMock),
+    getOffenderContacts: jest.fn(),
+    getCaseNoteSummaryByTypes: jest.fn(),
     getPrisoner: jest.fn(async () => prisonerDetailMock),
     getInmateDetail: jest.fn(async () => inmateDetailMock),
   }
 
+  const allocationManagerApiClient: AllocationManagerClient = {
+    getPomByOffenderNo: jest.fn(async () => pomMock),
+  }
+
+  const keyWorkerApiClient: KeyWorkerClient = {
+    getOffendersKeyWorker: jest.fn(async () => keyWorkerMock),
+  }
+
   beforeEach(() => {
     prisonApiClient.getNonAssociationDetails = jest.fn(async () => nonAssociationDetailsDummyData)
     prisonApiClient.getEventsScheduledForToday = jest.fn(async () => dummyScheduledEvents)
   })
 
   describe('Non-associations', () => {
     it.each(['ABC123', 'DEF321'])('Gets the non-associations for the prisoner', async (prisonerNumber: string) => {
-      const overviewPageService = new OverviewPageService(prisonApiClient)
+      const overviewPageService = new OverviewPageService(
```

Comment:
I wonder if we could just move the construction of the overview page service to at least a function - so line 62 would become:
```await service().get({prisonerNumber} as Prisoner)```
where `service()` is a function that returns
```
new OverviewPageService(prisonApiClient, ...)
```

Might clean up these tests a bit. Not sure if it'll work though